plugin-flex-ts-template-v2/src/feature-library/custom-hold-music/flex-hooks/events/pluginsInitialized.ts

```diff
@@ -0,0 +1,10 @@
+import { FlexEvent } from '../../../../types/feature-loader';
+import { getHoldMusicUrl } from '../../config';
+import ProgrammableVoiceService from '../../../../utils/serverless/ProgrammableVoice/ProgrammableVoiceService';
+
+export const eventName = FlexEvent.pluginsInitialized;
+export const eventHook = function setCommonPVServiceHoldUrl() {
+  // Some features may abort the HoldParticipant action and instead call the serverless service to hold instead
+  // This makes the serverless service also use the custom hold URL
+  ProgrammableVoiceService.holdUrl = getHoldMusicUrl();
```

Comment:
Just noting this as a possible improvement in a future update, not something to be addressed immediately. Would be preferable to update class properties through a setter instead of modifying the property directly. This could enable things like logging a warning if it's being set in multiple places and the developer isn't aware they're overwriting it.